dopamine/replay_memory/circular_replay_buffer.py

```diff
@@ -111,7 +111,8 @@ def __init__(self,
                action_dtype=np.int32,
                reward_shape=(),
                reward_dtype=np.float32,
-               lock=lock_lib.get_default_lock()):
+               lock=lock_lib.get_default_lock(),
```

Comment:
You found a simple solution, here, to a problem that could have seemed complicated.  Nice work.

Reply:
thanks